backend/sozisel/priv/repo/migrations/20210930175424_create_session_resources.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Sozisel.Repo.Migrations.CreateSessionResources do
+  use Ecto.Migration
+
+  def change do
+    create table(:session_resources) do
```

Comment:
You could do all those tables creation in a single migration but in fact I like it this way, you don't have to change anything here :)